### DISCUSSION TITLE
Can't bind IKEA E1743 controller to IKEA LED1624G9

### DISCUSSION THREAD

**Discussion by fribse** [Koenkk/zigbee2mqtt#14034] [Devices]
I've gotten quite a few binds up and running here, delightfull to work with compared to deconz, which I just switched away from.
But I have a problem with one light / controller.
It's a standard E1743 controller and a IKEA LED1624G9 which is a standard 600 lumens RGBW.
I can bind the controller and light with the OnOff and LevelCtrl clusters. But when I've done that, the light doesn't react to the controller.
I can make the light controllable through Home Assistant, just not directly with a bind?

I have several of these pairs, and it doesn't seem to work on any of those.

I have other IKEA types and I can do binds with them, no problem.

Any idea why?

**Comment by yurgh** [comment]
What firmware are the E1734 remotes running, maybe the remote can bind to groups only? https://www.zigbee2mqtt.io/devices/E1743.html#binding

**Comment by fribse** [comment]
@yurgh You were absolutely right, those two had failed to update to the latest firmware. After doing that, they can pair, thanks!
